scheduler/src/cook/mesos/api.clj

```diff
@@ -240,6 +240,11 @@
    (s/optional-key :constraints) [Constraint]
    (s/optional-key :container) Container
    (s/optional-key :executor) (s/enum "cook" "mesos")
+   (s/optional-key :executor-log-level) (s/enum "DEBUG" "ERROR" "INFO" "WARNING")
```

Comment:
Do we really want to let users modify this?

Reply:
@dposada @wyegelwel 

I introduced `executor-log-level` and `executor-max-message-length` as developer configs that can help us with debuggability. Both these values are sanitized and restricted to their allowed values in `task/build-executor-environment`.

Reply:
Id prefer dropping them but don't care that much. If you are going to keep them, please move the value checks to submission time and fail the submission if they are invalid so the user doesn't get any surprises.

Reply:
I'd prefer dropping them as well. If we find in the future that we're really missing them for debuggability, we can consider adding them

Reply:
Removed configurability of `executor-log-level`, `executor-max-message-length`, and `progress-sample-interval-ms`.